stock_location_flowable/models/stock_picking_type.py

```diff
@@ -0,0 +1,46 @@
+# Copyright NuoBiT Solutions - Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class PickingType(models.Model):
+    _inherit = "stock.picking.type"
+
+    flowable_operation = fields.Boolean()
+    count_picking_blocked = fields.Integer(compute="_compute_picking_count")
+
+    def _compute_picking_count(self):
+        res = super()._compute_picking_count()
+        for rec in self:
+            rec.count_picking_blocked = 1
+            # Se considera blqueado si el stock.picking.type es flowable_operation
+            # y la stock.location de origen es flowable y flowable_blocked es True
+            # rec.count_picking_blocked = self.env['stock.location'].search_count([
+            #     ('p')
+            # ])
+        return res
+
+    def get_action_picking_tree_blocked(self):
+        return self._get_action("stock.action_picking_tree_blocked")
+
+    @api.constrains("flowable_operation")
+    def _check_flowable_operation(self):
+        for rec in self:
+            if (
+                rec.env["stock.picking.type"].search_count(
+                    [
+                        ("flowable_operation", "=", True),
+                        ("warehouse_id", "=", rec.warehouse_id.id),
+                        ("company_id", "=", rec.company_id.id),
+                    ]
+                )
+                > 1
+            ):
+                raise UserError(
+                    _(
+                        "Only one picking type can be flowable filling in the warehouse %s."
```

Comment:
Replace by:
`Only one picking type can be `flowable` in a warehouse %s.`
And remember to update the `po`